sensor_msgs/PointCloud2[] point_clouds
---
uint8 status
pc_attitude_estimation/attitude_est[] AttitudeEstVec
